### PR TITLE
iOS Live Updates Integration

### PR DESCRIPTION
**Pull request by thomasvidas**

### CODE REVIEW

**Comment by thomasvidas** [comment]
Depends on https://github.com/ionic-team/live-updates-sdk/pull/16

File: ios/IonicPortals/IonicPortals/PortalWebView.swift [hunk 1]
```diff
@@ -22,10 +24,19 @@ public class PortalWebView: UIView {
...
+                self.liveUpdatePath = LiveUpdateManager.getLatestAppDirectory(liveUpdateConfig.getAppId())
+            }
+            webView = InternalCapWebView(frame: self.frame, portal: portal, liveUpdatePath: self.liveUpdatePath)
+            
+            guard let bridge = webView?.bridge else { return }
+            self.bridge = bridge
+            
             addSubview(webView!)
         } else {
+            print("444444444444444")
```
Comment (carlpoole) [inline ios/IonicPortals/IonicPortals/PortalWebView.swift:39]: 4444444444 :)

**Review by carlpoole — APPROVED** [review]
Looks good just the small log I think was being used for debug

File: ios/IonicPortals/IonicPortals/PortalWebView.swift (hunk 1 above)
Comment (thomasvidas) [reply ios/IonicPortals/IonicPortals/PortalWebView.swift:39]: Whoops 😅

**Pull request merged**